docs/examples/201/wvd-backplane-with-network-and-storage-and-monitoring/main.bicep

```diff
@@ -93,3 +93,16 @@ module wvdFileServices './wvd-fileservices-module.bicep' = {
     fileshareFolderName: fileshareFolderName
   }
 }
+
+//Create Private Endpoint for file storage
+module pep './wvd-fileservices-privateendpoint-module.bicep' = {
+  name: 'privateEndpoint'
+  scope: resourceGroup(rgnetw.name)
```

Comment:
the scope property now understands the symbolic resource name, so you can simplify this to:

```bicep
  scope: rgnetw
```

Reply:
@alex-frankel , yes that makes sense to add, thanks